.github/workflows/build.yml

```diff
@@ -40,22 +45,39 @@ jobs:
         with:
           name: libddwaf-windows-${{ matrix.arch }}
           path: ${{ github.workspace }}/packages/*.tar.gz
+          if-no-files-found: error
 
-  macos-build:
+  macos-builds:
     runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: x86_64
+            platform: x86_64-darwin
+            test: true
+          - arch: arm64
+            platform: arm64-darwin
+            test: false
+    name: macOS (${{ matrix.platform }})
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: recursive
       - name: Create Build Directory
         run: cmake -E make_directory ${{ github.workspace }}/build ${{ github.workspace }}/packages
       - name: Generating Build Scripts
-        run: cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DMACOSX_DEPLOYMENT_TARGET=10.9 -DCMAKE_INSTALL_PREFIX=${{ github.workspace }} -DCPACK_PACKAGE_DIRECTORY=${{ github.workspace }}/packages ${{ github.workspace }}
+        run: cmake -DCMAKE_OSX_ARCHITECTURES=${{ matrix.arch }} -DCMAKE_BUILD_TYPE=RelWithDebInfo -DMACOSX_DEPLOYMENT_TARGET=10.9 -DCMAKE_INSTALL_PREFIX=${{ github.workspace }} -DCPACK_PACKAGE_DIRECTORY=${{ github.workspace }}/packages ${{ github.workspace }}
```

Comment:
One thing I noticed with the current builds is that actually do not target 10.9, but rather 11.6:

```
$ curl -sL https://github.com/DataDog/libddwaf/releases/download/1.5.0/libddwaf-1.5.0-darwin-x86_64.tar.gz | tar -O - -xzf - libddwaf-1.5.0-darwin-x86_64/lib/libddwaf.dylib > /tmp/libddwaf.dylib; otool -l /tmp/libddwaf.dylib | grep minos
    minos 11.6
```

Reply:
IIRC I removed it either because of cross-compilation to arm64-darwin, which can't target below macOS 11 since that's the first version with `arm64-darwin` support or because the macOS CI runners were brownout'd and the updated ones refused to proceed with targeting macOS 10, but I can't recall which it was.

We can either make it a matrix variable if targeting 10.9 works for Intel (I'll try that), or drop support for macOS 10.

If the former works, I wonder if this value should be part of cmake instead of the CI config.

Reply:
Oh, wait, `-DMACOSX_DEPLOYMENT_TARGET=10.9` is still there. That might come from `runs-on: macos-latest` then?